pkg/controller/gitopsconfig/controller.go

```diff
@@ -391,17 +393,17 @@ func (r *Reconciler) manageDeletion(instance *gitopsv1alpha1.GitOpsConfig) (reco
 	jobList := &batchv1.JobList{}
 	selector, err := labels.Parse("action=delete")
 	if err != nil {
-		log.Error(err, "unable to parse label selector 'action=delete' ")
-		return reconcile.Result{}, err
+		log.Error(err, "unable to parse label selector 'action=delete'")
+		return reconcile.Result{}, xerrors.Errorf("unable to parse label selector 'action=delete': %w", err)
 	}
 	// looking up all delete jobs
 	err = r.client.List(context.TODO(), &client.ListOptions{
 		Namespace:     instance.GetNamespace(),
 		LabelSelector: selector,
 	}, jobList)
 	if err != nil {
-		log.Error(err, "unable to list jobs ")
-		return reconcile.Result{}, err
+		log.Error(err, "unable to list all delete jobs")
+		return reconcile.Result{}, xerrors.Errorf("unable to list all delete jobs: %w", err)
```

Comment:
on l. 180, there's an implicit "return err"; as a result, if this error happens, we won't know which instance it's related to; please either add this info in this error message, or expand l.180 to add it

Reply:
after discussion, can be left as is, thanks!